modules/local/spades.nf

```diff
@@ -1,55 +1,47 @@
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName } from './functions'
-
-params.options = [:]
-options    = initOptions(params.options)
-
 process SPADES {
```

Comment:
spades module might be not usable https://github.com/nf-core/modules/tree/master/modules/spades due to reproducibilty missing (params.spades_fix_cpus).